src/components/HeaderDropdown/index.less

```diff
@@ -1,3 +1,26 @@
+/*
+* MIT License
+
+* Copyright (c) 2019 Alipay.inc
```

Comment:
we should add Alipay to the LICENSE file, like https://github.com/apache/skywalking/blob/master/LICENSE#L204